test/test_wasm/test_wasm.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -e
+
+# this dir
+TEST_WASM_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+SRC_DIR=$TEST_WASM_DIR/../..
+
+
+git clone https://github.com/emscripten-core/emsdk
```

Comment:
this looks like an install step. Maybe split this script into `install_wasm_deps.sh` and `test_wasm.sh`? i assume there are no distribution package on ubuntu or mamba for emsdk.

Reply:
Yeah we can split that up.
We might want to use the emsdk from conda but for all other projects I just ended up cloning emsdk as it is not really designed to be packaged (is shipping things, has a strange directory structure, and so on)

Reply:
actually let's leave it as one script, otherwise it will be a tiny bit more complicated to pass a custom emsdk installation (as I do for local development)